samples/app.yml

```diff
@@ -3,10 +3,12 @@ templates:
   - "templates/redis.template.yml"
   - "templates/sidekiq.template.yml"
   - "templates/web.template.yml"
+  #- "templates/web.ssl.template.yml"
```

Comment:
This is there because it is optional, and in the new docs I'll explain they need to uncomment this line and the `expose 433:433` line.